upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -152,7 +156,49 @@ func (c *ApplyClusterCmd) Run() error {
 	}
 	c.channel = channel
 
-	assetBuilder := assets.NewAssetBuilder(c.Cluster.Spec.Assets)
+	stageAssetsLifecycle := fi.LifecycleSync
+	securityLifecycle := fi.LifecycleSync
+	networkLifecycle := fi.LifecycleSync
+	clusterLifecycle := fi.LifecycleSync
+
+	switch c.Phase {
+	case Phase(""):
+		stageAssetsLifecycle = fi.LifecycleIgnore
```

Comment:
Because we cannot check for assets, for now, skip them always.  Within air-gapped environments having this run all the time could really cause problems.  We can look at enabling this once we can actually check for the files and containers.

Reply:
Why can't we check for assets?   Probably we need a code comment here

Reply:
I think also this needs to be the same as when we can't see GCE ACLs

Reply:
It needs to be implemented for both containers and files.